### ISSUE TITLE
Class variables accessed via intermedate variable flagged as unused

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1150]
I dislike noise, visual designer code is littered with ``this-object:``, so I added an intermediate variable ``o`` and assigned it ``this-object``. All variables were then flagged as unused.
````
class foo:

   var integer bar.
   var integer bar-this-object.
   var integer bar-o. // <-- flagged as unused

   constructor foo ():

      var foo o.

      bar = 1.
      message bar.

      this-object:bar-this-object = 2.
      message this-object:bar-this-object.

      o = this-object.
      o:bar-o = 3.
      message o:bar-o.

   end constructor.

end class.
````

**Comment by gquerret** [comment]
Confirmed, will be fixed

**Comment by gquerret** [comment]
Fixed as part of #591
